### PR TITLE
Add nfs-server-provisioner helm chart

### PR DESCRIPTION
**Pull request by kvaps**
Moving helm chart from [stable/nfs-server-provisioner](https://github.com/helm/charts/tree/master/stable/nfs-server-provisioner) and [kvaps/nfs-server-provisioner-chart](https://github.com/kvaps/nfs-server-provisioner-chart) repositories.

All the history of commits is preserved

### CODE REVIEW

**Comment by kvaps** [comment]
Not sure If I can sign off for the others

**Comment by kmova** [comment]
@kvaps - CLA is tricky. So for the migration PRs from kubernetes-incubator to this new repo, I had to cherry pick the commits and fix the ones without CLA by dropping them and adding a new commit with the changes.

**Comment by kvaps** [comment]
> @kvaps - CLA is tricky. So for the migration PRs from kubernetes-incubator to this new repo, I had to cherry pick the commits and fix the ones without CLA by dropping them and adding a new commit with the changes.

Okay seems rebased, any idea if `/deploy/helm` good place for storing the helm chart?

File: deploy/helm/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  email: kiall@macinnes.ie
+- name: kvaps
+  email: kvapss@gmail.com
+- name: joaocc
+  email: joaocc-dev@live.com
+- name: naseemkullah
+  email: naseem@transit.app
+home: https://github.com/kubernetes/charts/tree/master/stable/nfs-server-provisioner
+sources:
+- https://github.com/kubernetes-incubator/external-storage/tree/master/nfs
```
Comment (consideRatio) [inline deploy/helm/Chart.yaml]:
```suggestion
home: https://github.com/kubernetes-sigs/nfs-ganesha-server-and-external-provisioner
sources:
- https://github.com/kubernetes-sigs/nfs-ganesha-server-and-external-provisioner/tree/master/deploy/helm
```

File: deploy/helm/README.md [hunk 2]
```diff
@@ -0,0 +1,206 @@
+# NFS Server Provisioner
+
+[NFS Server Provisioner](https://github.com/kubernetes-incubator/external-storage/tree/master/nfs)
```
Comment (consideRatio) [inline deploy/helm/README.md]:
```suggestion
[NFS Server Provisioner](https://github.com/kubernetes-sigs/nfs-ganesha-server-and-external-provisioner)
```

File: deploy/helm/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+maintainers:
+- name: kiall
+  email: kiall@macinnes.ie
+- name: kvaps
+  email: kvapss@gmail.com
+- name: joaocc
+  email: joaocc-dev@live.com
+- name: naseemkullah
+  email: naseem@transit.app
+home: https://github.com/kubernetes/charts/tree/master/stable/nfs-server-provisioner
```
Comment (kmova) [inline deploy/helm/Chart.yaml]: @kvaps - wondering if we can get these links updated in the current PR itself as an additional commit. Also update the references from kubernetes-incubator to kubernetes-sigs.

**Comment by kmova** [comment]
Thanks @kvaps - this looks good. Just had one minor comment. Let me know if you would like to take that up as part of this commit or in another PR.

**Comment by consideRatio** [comment]
Thank you for your work on this @kvaps and @kmova!

**Pull request merged**